index.d.ts

```diff
@@ -0,0 +1,13 @@
+import unified from 'unified'
+import unist from 'unist'
+import hast from 'hast'
+
+
+interface RemarkAutolinkHeadingsOptions {
+  behavior?: 'prepend' | 'append' | 'wrap' | 'before' | 'after'
+  linkProperties?: Record<string, any>
+  content?: any
+  group?: (node: unist.Node) => hast.Node
+}
+
+export default function(options?: RemarkAutolinkHeadingsOptions): unified.Transformer
```

Comment:
I believe it’s typically preferred to use the higher level type `unified.Plugin`.

```suggestion
declare const remarkAutolinkHeadings: Plugin<[RemarkAutolinkHeadingsOptions?]>
export default remarkAutolinkHeadings
```

However, it’s just a small suggestion. I’m fine with keeping this as-is.